src/app/Components/InputForm.jsx

```diff
@@ -1,92 +1,137 @@
 import { useState } from 'react';
 import "../styles/inputForm-styles.css"
-
+import axios from 'axios';
+import Cookies from 'js-cookie';
 function InputForm() {
```

Comment:
Make sure that after "Submitting the form" or "clicking the reset button" all the form data is cleared along with the data present in the text-area.